### ISSUE TITLE
Transparent App icon on macos is difficult to hit

### ISSUE THREAD

**Issue by bortels** [thonny/thonny#1530]
Macs are a bit brain damaged, in that the transparent areas on the icon for an app are *not* clickable. 

That means to launch the Thonny app, you have to hit on the thin bars of the "Th" letters that the app icon has - the empty space between or inside the characters isn't a valid target. 

I'd suggest an opaque background to make it easier to launch.

**Comment by r0the** [comment]
@bortels Good idea, an icon with background would also better match other application icons in the dock. Currently it sticks out a bit:

<img width="790" alt="Bildschirmfoto 2021-01-11 um 17 28 16" src="https://user-images.githubusercontent.com/1057839/104209957-9a5fce00-5432-11eb-9f38-5851ece4a563.png">

@aivarannamaa Here's a proposal based on the current app icon [here](https://github.com/thonny/thonny/blob/master/thonny/res/thonny.png) and the  [Apple Design Resources](https://developer.apple.com/design/resources/#macos-apps).

![Thonny Icon macOS 11](https://user-images.githubusercontent.com/1057839/104211453-61c0f400-5434-11eb-8f52-c61c616578da.png)

Preview in dock:

<img width="757" alt="Bildschirmfoto 2021-01-11 um 17 41 51" src="https://user-images.githubusercontent.com/1057839/104211393-5241ab00-5434-11eb-96ce-136f753c5ee2.png">

But it's very white. Maybe either background should be a different colour or the letters should be filled or be
in a different colour...

**Comment by aivarannamaa** [comment]
Thank you both for report and suggestion. I never noticed this problem myself.

I think the white one is perfectly good. Thanks!

**Comment by r0the** [comment]
Actually, the non-clickable transparency is only a problem in the finder when the files are displayed as icons. In the dock and launcher, the transparent areas are clickable.

**Comment by aivarannamaa** [comment]
I shifted the letters a bit, so that it looks a bit more balanced:
![thonny_mac](https://user-images.githubusercontent.com/1802/158617658-922de69e-18f5-4cc1-9b50-eaac507086f1.png)

**Comment by aivarannamaa** [comment]
> But it's very white. Maybe either background should be a different colour or the letters should be filled or be in a different colour...

@r0the, what do you think of making the background partly transparent?

**Comment by varunkumarmedam** [comment]
Any work around for making transparent background clickable?
